articles/101_build_tool.md

```diff
@@ -55,16 +65,19 @@ In the ROS ecosystem [rosdep](http://wiki.ros.org/rosdep) can be used for this.
 The build tool also does not create binary packages (e.g. a Debian package).
 In the ROS ecosystem [bloom](http://wiki.ros.org/bloom) is used to generate the required metadata and then platform dependent tools like `dpkg-buildpackage` build binary packages.
 
-## Build Tool vs. Build System
+## Build Tool vs. Build System vs. Package manager
```

Comment:
Since this article isn't about package managers I don't see why this should be added here. Maybe move it to the "Out of scope" section instead.